scipy/stats/tests/test_multivariate.py

```diff
@@ -64,6 +64,15 @@ def test_logpdf(self):
         d2 = multivariate_normal.pdf(x, mean, cov)
         assert_allclose(d1, np.log(d2))
 
+    def test_logpdf_default_values(self):
+        # Check that the log of the pdf is in fact the logpdf
+        # with default parameters Mean=None and cov = 1
+        np.random.seed(1234)
+        x = np.random.randn(5)
+        d1 = multivariate_normal.logpdf(x)
+        d2 = multivariate_normal.pdf(x)
+        assert_allclose(d1, np.log(d2))
```

Comment:
Here you test only that ``pdf`` and ``logpdf`` have consistent defaults. Probably you also want to test that the defaults are what they should be. So test ``(x)`` against ``(x, None, 1)``